packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openstack/openstackSecretFieldValidator.ts

```diff
@@ -270,19 +358,19 @@ const validateApplicationCredentialName = (value: string): ValidationMsg => {
 };
 
 const validateInsecureSkipVerify = (value: string): ValidationMsg => {
-  const valid = ['true', 'false', ''].includes(value);
+  const valid = ['true', 'false', '', undefined].includes(value);
```

Comment:
maybe undefined should be `default` and not `success` ?

Reply:
Correct. We are not displaying the `helperText` for insecureSkipVerify fields, only the `helperTextInvalid`, so it's not that important but I'll fix that